sql/catalyst/src/main/scala/org/apache/spark/sql/connector/catalog/LookupCatalog.scala

```diff
@@ -130,39 +163,12 @@ private[sql] trait LookupCatalog extends Logging {
    */
   object AsTemporaryViewIdentifier {
```

Comment:
It seems this object is only used in LookupCatalogSuite. if it is only for test, shall we put it in LookupCatalogSuite.scala ?

Reply:
if it's only used in test, we can remove it.

Reply:
sure. I can do a quick follow-up PR.